File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_nestbr.py
"""Reference emit for NESTBR — nested IF grade bands, score 75 → 2."""
# EXPECTED: 2
SCORE = 75
if SCORE >= 90:
    GRADE = 4
elif SCORE >= 80:
    GRADE = 3
elif SCORE >= 70:
    GRADE = 2
else:
    GRADE = 1
print(GRADE)

File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_nestbr.java
// EXPECTED: 2
// Reference emit for NESTBR — nested IF grade bands.
public final class NestbrRef {
  public static void main(String[] args) {
    int score = 75;
    int grade;
    if (score >= 90) grade = 4;
    else if (score >= 80) grade = 3;
    else if (score >= 70) grade = 2;
    else grade = 1;
    System.out.println(grade);
  }
}
